src/Lykke.Service.Dash.Api/Controllers/TransactionsController.cs

```diff
@@ -0,0 +1,252 @@
+        }
+
+        [HttpPost("broadcast")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<IActionResult> Broadcast([Required, FromBody] BroadcastTransactionRequest request)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ErrorResponse.Create("ValidationError", ModelState));
+            }
+
+            var broadcast = await _dashService.GetBroadcastAsync(request.OperationId);
+            if (broadcast != null)
+            {
+                return new StatusCodeResult(StatusCodes.Status409Conflict);
+            }
+
+            var transaction = _dashService.GetTransaction(request.SignedTransaction);
+            if (transaction == null)
+            {
+                return BadRequest(ErrorResponse.Create($"{nameof(request.SignedTransaction)} is not a valid"));
+            }
+
+            await _log.WriteInfoAsync(nameof(TransactionsController), nameof(Broadcast),
+                request.ToJson(), "Broadcast transaction");
+
+            await _dashService.BroadcastAsync(transaction, request.OperationId);
+
+            return Ok();
+        }
+
+        [HttpGet("broadcast/single/{operationId}")]
+        [ProducesResponseType(typeof(BroadcastedSingleTransactionResponse), StatusCodes.Status200OK)]
+        public async Task<IActionResult> GetBroadcast([Required] Guid operationId)
+        {
+            var broadcast = await _dashService.GetBroadcastAsync(operationId);
+            if (broadcast == null)
+            {
+                return NoContent();
+            }
+
+            var amount = broadcast.Amount.HasValue ?
+                Conversions.CoinsToContract(broadcast.Amount.Value, Asset.Dash.Accuracy) : "";
+
+            var fee = broadcast.Fee.HasValue ?
+                Conversions.CoinsToContract(broadcast.Fee.Value, Asset.Dash.Accuracy) : "";
+
+            return Ok(new BroadcastedSingleTransactionResponse
+            {
+                OperationId = broadcast.OperationId,
+                Hash = broadcast.Hash,
+                State = broadcast.State.ToBroadcastedTransactionState(),
+                Amount = amount,
+                Fee = fee,
+                Error = broadcast.Error,
+                Timestamp = broadcast.GetTimestamp(),
+                Block = broadcast.Block
+            });
+        }
+
+        [HttpDelete("broadcast/{operationId}")]
+        public async Task<IActionResult> DeleteBroadcast([Required] Guid operationId)
+        {
+            var broadcast = await _dashService.GetBroadcastAsync(operationId);
+            if (broadcast == null)
+            {
+                return NoContent();
+            }
+
+            await _log.WriteInfoAsync(nameof(TransactionsController), nameof(DeleteBroadcast),
+                new { operationId = operationId }.ToJson(), 
+                "Delete broadcast");
+
+            await _dashService.DeleteBroadcastAsync(broadcast);
```

Comment:
If execution fail after this line (or after data has gone broadcast repositort), data from build repository will never be deleted